sherpa-onnx/c-api/c-api.h

```diff
@@ -491,7 +496,9 @@ SHERPA_ONNX_API void DecodeMultipleOfflineStreams(
 SHERPA_ONNX_API typedef struct SherpaOnnxOfflineRecognizerResult {
   const char *text;
 
-  // Pointer to continuous memory which holds timestamps
+  const char *lang;
```

Comment:
For the newly added member, please add it as the last field.

That is, move line 499 to line 529

Reply:
Done.